ansible/group_vars/all

```diff
@@ -125,8 +125,9 @@ kafka:
   version: 0.10.2.1
   port: 9092
   ras:
-    port: 9093
+    port: 8093
   heap: "{{ kafka_heap | default('1g') }}"
+  replicationFactor: 1
```

Comment:
The `replicationFactor` is an important setting for the resiliency of the kafka cluster. If it's set to `1` by default the system would not be resilient against failures of a kafka node.

Therefore I would prefer to handle the `replicationFactor` in a way that we can override it per environment similar to the `heap` setting above, i.e.

```
replicationFactor: {{ kafka_replicationFactor | default(1) }}
```

In a particular environment where we use multiple kafka nodes we would then be able to override the replicationFactor accordingly. Also, I think that's in general a good practice in our ansible playbooks.